spec/json_matchers/match_json_schema_spec.rb

```diff
@@ -59,6 +59,147 @@
     expect(json).to match_json_schema("api/v1/schema")
   end
 
+  it "supports invalidating the referenced schema when using local references" do
+    create(:schema, name: "post", json: {
+      "$schema": "https://json-schema.org/draft-04/schema#",
+      "id": "file:/post.json#",
+      "definitions": {
+        "attributes": {
+          "type": "object",
+          "required": [
+            "id",
+            "name",
+          ],
+          "properties": {
+            "id": { "type": "string" },
+            "name": { "type": "string" }
+          }
+        }
+      },
+      "type": "object",
+      "required": [
+        "id",
+        "type",
+        "attributes"
+      ],
+      "properties": {
+        "id": { "type": "string" },
```

Comment:
@musaffa I've changed to this `"string"` according to https://github.com/thoughtbot/json_matchers/pull/89#discussion_r231100974.